src/src/components/drawer/Drawer.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+<div class="drawer-container"><!-- Sidebar -->
```

Comment:
Codacy has a fix for the issue: [Expected 1 line break after opening tag (`<div>`), but no line breaks found.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
<div class="drawer-container">
```